File: beecrowd/1046 - Tempo de Jogo.go
package main

import "fmt"

func main() {
	var h1, h2 int32

	fmt.Scan(&h1, &h2)

	var time int32

	if h1 < h2 {
		time = h2 - h1
	} else {
		time = 24 - h1 + h2
	}

	fmt.Printf("O JOGO DUROU %d HORA(S)\n", time)
}

File: beecrowd/1046 - Tempo de Jogo.kt
import java.util.*

fun main(args: Array<String>) {

    val sc = Scanner(System.`in`)

    sc.useLocale(Locale.US)

    val horaInicio = sc.nextInt()
    val horaFinal = sc.nextInt()

    val horas = if(horaInicio < horaFinal) {
        horaFinal - horaInicio
    } else {
        24 - horaInicio + horaFinal
    }

    println("O JOGO DUROU $horas HORA(S)")
}